api/services/contact_fields_mapping_service.py

```diff
@@ -27,7 +29,9 @@ def handle_contact_fields_data(self, jsonData):
                             contact_field_mapping, user_details, field_value
                         )
         except Exception as e:
-            print(f"Error occurred while handling custom field mapping. Exception: {e}")
+            logger.error(
+                f"Error occurred while handling custom field mapping for {self.user_phone}. Exception: {e}"
```

Comment:
Lets make the language consistent
```suggestion
                f"Error occurred while handling custom field mapping for {self.user_phone}. Error message: {e}"
```